send_to_telemetry.py

```diff
@@ -0,0 +1,104 @@
+from dataclasses import asdict, dataclass
+import json
+import logging
+import os
+import sys
+import uuid
+
+from jsonschema import validate
+
+import requests
+
+
+@dataclass
+class TestResult:
+    appName: str
+    channel: str
+    version: str
+    connection: str
+    url: str
+    platform: str
+    runner: str
+    runId: str
+    sessionState: str
+    metrics: dict
+
+
+def main(path):
+    with open(path) as f:
+        data = json.load(f)
+
+    with open("metrics.json") as f:
+        metrics = json.load(f)
+
+    for test in data:
+        # create an empty dictionary
+        values = {}
+        # grab each metric from metrics.json
+        for metric in metrics:
+            name = metric["name"]
+            # now, grab all metrics' values
+            for measure in ["median", "standardDeviation"]:
+                sample = test["data"][measure]["firstView"].get(name)
+                # sefdefault()s here will return each metric name & value,
+                # or return and set an empty dict
+                if sample is not None:
+                    m = values.setdefault(name, {})
+                    # write each metric having a "firstView" entry + value,
+                    # or return and set an empty dict
+                    first_view = m.setdefault("firstView", {})
+                    first_view[measure] = sample
+
+        sample = test["data"]["median"]["firstView"]
+        # get browser name and channel, then lower()-case them
+        browser, _, channel = sample["browser_name"].lower().partition(" ")
+
+        result = TestResult(
+            appName=browser,
+            channel=channel or "release",
+            version=sample["browser_version"],
+            connection=test["data"]["connectivity"].lower(),
+            url=test["data"]["testUrl"],
+            platform="desktop",
+            runner=os.getenv("JENKINS_URL", "unknown"),
+            runId=os.getenv("BUILD_TAG", "unknown"),
+            sessionState="noAuth",
+            metrics=values,
```

Comment:
`values` is a `dict` of `dicts`.  it is best if it is flattened so it is easier to process with SQL.

For each leaf in the dict tree emit a `TestResult`:

```python
results = [ 
    TestResult(
        ...properties, like above...
        metric_name=metric_name,
        measure=measure,
        sample=sample  # not sure what this sample is, I assume it is a number
    }
    for metric_name, measures in values.items()
    for measure, sample in measures.items()
]
```

Well, you may want to scrub

Reply:
I promise to follow up on this later, and/or could you soon submit a PR once _this_ PR lands?  (Which is eminent!)